src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -329,9 +248,8 @@ void Terminal::ClearSelection()
 {
     _selectionActive = false;
     _allowSingleCharSelection = false;
-    _selectionAnchor = { 0, 0 };
-    _endSelectionPosition = { 0, 0 };
-    _selectionVerticalOffset = 0;
+    _selectionStart = { 0, 0 };

```

Comment:
I do not love how we have a boolean _and_ a coord.. that's just an std::optional in disguise lol

Reply:
do not feel compelled to fix this for this pr

Reply:
I made the change in the latest commit. Lemme know what you think. If you don't like it or wanna go a different direction, I can just revert that one 😀

Reply:
Commit: https://github.com/microsoft/terminal/pull/4560/commits/12ba9eeb8f64c0f1cd6ee2eb07ae8c9200fd23c4